drawer-view/DrawerView.swift

```diff
@@ -402,3 +407,19 @@ private extension DrawerView {
         sholdRecalculateConstraints = false
     }
 }
+
+extension DrawerView : UIGestureRecognizerDelegate {
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
+        for subView in self.subviews {
```

Comment:
One small thing to note is the naming style for `subView` and `subviews`. Since the instance property is `subviews` without camel case, it's better to stick with this kind of naming convention for `subVIew` and instead use `subview`, in order to be more consistent. 

This is a minor thing, I will make this change after the PR 😉